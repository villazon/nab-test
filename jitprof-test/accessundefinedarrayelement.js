var a = [];
a[3] = 0;
a[2] = a[0] + a[1];
