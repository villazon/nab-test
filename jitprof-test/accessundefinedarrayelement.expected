---------------------------
Report of loading undeclared or deleted array elements:
 * [location: (jitprof-test/accessundefinedarrayelement.js:3:8:3:12)] <- No. usages: 1
 * [location: (jitprof-test/accessundefinedarrayelement.js:3:15:3:19)] <- No. usages: 1
...
Number of loading undeclared or deleted array elements spotted: 2
[****]AccessUndefArrayElem: 2
